src/views/dialogs/ShareAffiliateDialog.tsx

```diff
@@ -45,6 +49,14 @@ export const ShareAffiliateDialog = ({ setIsOpen }: DialogProps<ShareAffiliateDi
     affiliateMaxEarningQuery: { data: maxEarningData },
   } = useAffiliatesInfo(dydxAddress);
 
+  useEffect(() => {
+    if (data?.isEligible === undefined) return;
+
+    track(
+      AnalyticsEvents.AffiliateInviteFriendsModalOpened({ isAffiliateEligible: data.isEligible })
+    );
```

Comment:
we already track modal opening and closing in useAnalytics. If we combine that event with an event that fires on isEligible we could make the same funnel without duplicating analytics events.

Reply:
we have to track ineligible ones too 😭